### PR TITLE
update nuget packages to latest preview releases

### PR DESCRIPTION
**Pull request by networkfusion**
However, we should be mindful that people are now using stable releases (not that it particularly matters in this instance as the packages were already previews. Perhaps we should have 2 branches for both stable and preview???

### CODE REVIEW

File: Examples/Device/Device.SmallMemory.nanoFramework/app.config [hunk 1]
```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (josesimoes) [inline Examples/Device/Device.SmallMemory.nanoFramework/app.config]: We don't support app.config for assembly binding. If VS added this one here, something is wrong with a referenced assembly.

File: Examples/Device/Device.Thermometer.nanoFramework/app.config [hunk 2]
```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (josesimoes) [inline Examples/Device/Device.Thermometer.nanoFramework/app.config]: same here

File: Examples/ServiceBus/ServiceBus.EventHub.nanoFramework/app.config [hunk 3]
```diff
@@ -4,7 +4,7 @@
     <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
```
Comment (josesimoes) [inline Examples/ServiceBus/ServiceBus.EventHub.nanoFramework/app.config]: and here

File: nanoFramework/Amqp.Micro.nanoFramework.nfproj [hunk 4]
```diff
@@ -104,38 +104,52 @@
     </Compile>
   </ItemGroup>
   <ItemGroup>
+    <Reference Include="mscorlib">
```
Comment (josesimoes) [inline nanoFramework/Amqp.Micro.nanoFramework.nfproj]: here's the reason the app.config: there a duplicate entry for mscorlib

File: Examples/Device/Device.Thermometer.nanoFramework/Device.Thermometer.nanoFramework.nfproj [hunk 5]
```diff
@@ -26,28 +26,42 @@
     <ProjectReference Include="..\..\..\nanoFramework\Amqp.nanoFramework.nfproj" />
   </ItemGroup>
   <ItemGroup>
+    <Reference Include="mscorlib">
```
Comment (josesimoes) [inline Examples/Device/Device.Thermometer.nanoFramework/Device.Thermometer.nanoFramework.nfproj]: here's the reason the app.config: there a duplicate entry for mscorlib

File: nanoFramework/app.config [hunk 6]
```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (josesimoes) [inline nanoFramework/app.config]: same here

File: Examples/Device/Device.SmallMemory.nanoFramework/app.config (hunk 1 above)
Comment (networkfusion) [reply Examples/Device/Device.SmallMemory.nanoFramework/app.config]: Removed

File: Examples/Device/Device.Thermometer.nanoFramework/Device.Thermometer.nanoFramework.nfproj (hunk 5 above)
Comment (networkfusion) [reply Examples/Device/Device.Thermometer.nanoFramework/Device.Thermometer.nanoFramework.nfproj]: Removed

File: Examples/Device/Device.Thermometer.nanoFramework/app.config (hunk 2 above)
Comment (networkfusion) [reply Examples/Device/Device.Thermometer.nanoFramework/app.config]: Removed

File: Examples/ServiceBus/ServiceBus.EventHub.nanoFramework/app.config (hunk 3 above)
Comment (networkfusion) [reply Examples/ServiceBus/ServiceBus.EventHub.nanoFramework/app.config]: Removed

File: nanoFramework/Amqp.Micro.nanoFramework.nfproj (hunk 4 above)
Comment (networkfusion) [reply nanoFramework/Amqp.Micro.nanoFramework.nfproj]: Fixed

File: nanoFramework/app.config (hunk 6 above)
Comment (networkfusion) [reply nanoFramework/app.config]: Fixed

**Comment by networkfusion** [comment]
I am still looking for why it is looking to mscorlib 1.5.1 on builds... Getting there slowly!

**Comment by networkfusion** [comment]
@josesimoes I am stuck on the nugetspec files... I cant see anything wrong personally, but there must be!

**Comment by networkfusion** [comment]
@josesimoes , that was harder than I thought it would be... But builds successfully now!

**Comment by josesimoes** [comment]
If you can squash all these commits, that would be perfect.

**Comment by networkfusion** [comment]
Can you not just "Squash and Merge"?

**Comment by josesimoes** [comment]
Yes and will work for the merge. Was just trying that the PR would look neat. 😉

**Pull request merged**